imports/message-read-receipt/server/hooks.js

```diff
@@ -13,7 +13,7 @@ callbacks.add('afterSaveMessage', (message, room) => {
 	Subscriptions.setAsReadByRoomIdAndUserId(room._id, message.u._id);
 
 	// mark message as read as well
-	ReadReceipt.markMessageAsReadBySender(message, room._id, message.u._id);
+	ReadReceipt.markMessageAsReadBySender(message, room._id, message.u._id, message.token);
```

Comment:
This is not a good approach.
The `token` needs to come from a specific Livechat Room Type method.
In addition, the `markMessageAsReadBySender` needs to expect an object, not a regular variable.